src/test/features/createCampaign.feature

```diff
@@ -0,0 +1,23 @@
+@campaign
+Feature: Report
+
+Background: 
+ Given User passes the authorization on "Auth Page"
+
+ Scenario: Generate a new report
```

Comment:
I could not generate a report by following the steps in your scenario. Are you sure those steps are for generating a report?

Reply:
Yes, it for definitely for generating a new report on Reports page

Reply:
But your scenario steps are for creating a campaign.